rmgpy/reduction/reduction.py

```diff
@@ -268,13 +268,18 @@ def searchTargetIndex(targetLabel, reactionModel):
     """
     Searches for the Species object in the core species
     of the reaction that has the same label as the parameter string.
-    """
+    reactionModel must be of class CoreEdgeReactionModel
 
+    Has known issues dealing with duplicate labels. See reductionTest.py
+    for a unittest of this issue.
+    """
+    spc_labels = []
     for i, spc in enumerate(reactionModel.core.species):
         if spc.label == targetLabel:
             return i
+        spc_labels.append(spc.label)
 
-    raise Exception('{} could not be found...'.format(targetLabel))
+    raise Exception('{} could not be found in species labels {}.'.format(targetLabel,spc_labels))
```

Comment:
sure, let's revert to the previous exception, and focus on the documentation part which is actually well written.